pages/juan-ortizc/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body class="carousel">
+    <header class="header-carousel">
+      <h1 class="tittle">OUR <span class="tittle__orange">HOTELS</span></h1>
```

Comment:
I think that the class used for the h1 could be improved a little bit for example H1.slider__title and the modified part of the title maybe use something more as slider__title--orange. Remember the BEM convention Block - Slider, Element - Title, Modifier - Title - Orange